magpie/api/exception.py

```diff
@@ -27,7 +27,7 @@
 
 
 # noinspection PyPep8Naming
-def verify_param(   # noqa: E126
+def verify_param(   # noqa: E126,N802
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Argument name "ofType" doesn't conform to snake_case naming style](https://app.codacy.com/app/fmigneault/Magpie/pullRequest?prid=4650806)